src/node.cc

```diff
@@ -3803,6 +3810,31 @@ static void ParseArgs(int* argc,
       }
       args_consumed += 1;
       config_userland_loader = module;
+    } else if (strncmp(arg, "--mode=", 7) == 0) {
+      if (!config_experimental_modules) {
+        fprintf(stderr, "%s: %s requires --experimental-modules be enabled\n",
+            argv[0], arg);
+        exit(9);
+      }
+      config_main_mode = arg + 7;
+      if (config_main_mode != "esm") {
+        fprintf(stderr, "%s: \"%s\" is not a valid module mode\n", argv[0],
+            config_main_mode.c_str());
+        exit(9);
+      }
+    } else if (strcmp(arg, "--mode") == 0) {
+      const char* main_mode = argv[index + 1];
+      if (main_mode == nullptr) {
+        fprintf(stderr, "%s: %s requires an argument\n", argv[0], arg);
+        exit(9);
+      }
+      args_consumed += 1;
+      config_main_mode = main_mode;
+      if (config_main_mode != "esm") {
+        fprintf(stderr, "%s: \"%s\" is not a valid module mode\n", argv[0],
```

Comment:
If there is two modes documented shouldn't both modes be allowed?

Reply:
it does seem like you should be able to set "commonjs" or "cjs" as the mode